pygeo/parameterization/DVGeo.py

```diff
@@ -1520,6 +1520,7 @@ def addCompositeDV(self, dvName, ptSetName=None, u=None, scale=None, prependName
             self.computeTotalJacobian(ptSetName)
             J_full = self.JT[ptSetName].todense()  # this is in CSR format but we convert it to a dense matrix
             u, s, _ = np.linalg.svd(J_full, full_matrices=False)
+            u = np.array(u)
```

Comment:
The other two changes look good. Can you explain this one a bit more? I thought `u` was supposed  to be an `N_DV` by `N_DV` matrix

Reply:
Yeah it is, the issue is that it's an N_DV by N_DV `np.matrix` not `np.array`. Because of this, `values` ends up being a (1,265) `np.matrix` and that breaks pyoptsparse. Calling flatten on a matrix doesn't do anything because the matrix type always wants to remain 2d.

Reply:
@ArshSaja , I think you're the only person other than @ewu63 who's used the composite DVs, did you ever run into this?

Reply:
Oh, good catch, shows you how observant I am. I agree, will change to `toarray` and remove this conversion

Reply:
@ArshSaja confirmed that he also had to fix this to use composite DVs, so it seems like this was broken from the start

Reply:
I had similar issue in the past. Since this is local change (`todense()` to `toarray()` in that function, it won't affect anywhere else I think. It looks good to me